scripts/composer/post-install.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/sh
 
+DOCUMENTROOT=web
+
+# Prepare the scaffold files if they are not already present
+if [ ! -f $DOCUMENTROOT/index.php ]
+  then
+    composer drupal-scaffold
+    mkdir -p $DOCUMENTROOT/modules
```

Comment:
Should these mkdir's be a part of the drupal-scaffold command? If I run `composer drupal-scaffold`, I think I'd expect the resulting web dir to be pretty much identical to what the core tarball ships.